site/content/en/docs/tasks/run/multikueue/job.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Run Kubernetes Job in Multi-Cluster"
+linkTitle: "Kubernetes Job"
+weight: 2
+date: 2024-11-05
+description: >
+  Run a MultiKueue scheduled Kubernetes Job.
+---
+
+## Before you begin
+
+Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+For the ease of setup and use we recommend using at least Kueue v0.8.1 and Kubernetes v1.30.0.
+
+## Run Kubernetes Job example
+
+Once the setup is complete you can test it by running the [`job-sample.yaml`](/docs/tasks/run/jobs/#1-define-the-job) Job. 
+
+The recommended way of running MultiKueue depends on the configuration of the `JobManagedBy` feature gate in your cluster. 
+
+NOTE: The `JobManagedBy` feature gate is disabled in 1.30 and 1.31 by default, and will be enabled in 1.32 by default.
+
+## Cluster with JobManagedBy enabled
+
+When `JobManagedBy` is enabled in your cluster we recommend configuring Kueue to enable the `MultiKueueBatchJobWithManagedBy` feature gate. 
+
+When `MultiKueueBatchJobWithManagedBy` is enabled Kueue defaults the `spec.managedBy` field to `kueue.x-k8s.io/multikueue`. This allows to disable processing of the Job by the built-in controller, and allows MultiKueue
+to send live updates reflecting the status of the mirror Job on the worker.
```

Comment:
It seems that this is an internal specification. What is the benefit for the batch users?
I guess that the users can observe the actual Job state via Management cluster, right?

Is there anything else?

Reply:
> I guess that the users can observe the actual Job state via Management cluster, right?

Yes, but this is very important. Basically we want to make MultiKueue transparent to the end-users. 
Note that there might be many worker clusters. Without the feature, the user needs to check the current status a user needs to inspect the workload object to identify which worker is running the Job.
Then, lookup the Job on the worker cluster, this is tedious for manual job. 

Also it is very important for automations. For example some companies may have in-house built UIs to track the Job progress. Without the feature they would need to complicate the code of the integrations and make 
it MultiKueue-aware. With this feature the UI code does not need any adjustments.